src/main/webapp/script.js

```diff
@@ -44,7 +44,7 @@ function renderSingleTask(task) {
  * reports if there are problems.
  */
 function updateView(results) {
-  // TODO: show how many tasks were scheduled, notify about conflicts/problems
+  $("#message").html(results.message);
```

Comment:
We could easily make the UI better by using some existing front-end library. (That has for example some built-in notifications) At the moment I think it is not a high priority, but still I'm curious for your suggestions.

Reply:
Let's do this in the _Polish UX_ step after we have the MVP working.